src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/NotificationReferenceBuilder.java

```diff
@@ -200,4 +200,18 @@ public static String referenceForDefendant(String claimReferenceNumber) {
             return reference("settlement-rejected-%s", DEFENDANT, claimReferenceNumber);
         }
     }
+
+    public static class LegalOrderDrawn {
+        private LegalOrderDrawn() {
+            // do not instantiate
+        }
+
+        public static String referenceForClaimant(String claimReferenceNumber) {
+            return reference("to-%s-legal-order-drawn-%s", CLAIMANT, claimReferenceNumber);
+        }
+
+        public static String referenceForDefendant(String claimReferenceNumber) {
```

Comment:
Both of these method can be replaced with single method with additional parameter for party. Please also add unit tests for this.

Reply:
👍 was probably following the existing pattern a bit mindlessly, will take a look

Reply:
I was testing it in `OrderDrawnNotificationServiceTest` but having a unit test closer to the code is always good :D added